lib/ecto/parameterized_type.ex

```diff
@@ -135,7 +138,7 @@ defmodule Ecto.ParameterizedType do
   @doc """
   Checks if two terms are semantically equal.
   """
-  @callback equal?(value1 :: any(), value2 :: any(), params :: params()) :: boolean()
+  @callback equal?(value1 :: any(), value2 :: any(), params :: params()) :: boolean() | :ignore
```

Comment:
The  `:ignore` (or `:skip`) should be in `change`. Please see the comment I posted on Ecto.Type for a reference implementation. :)

Reply:
done